server/backend/check.ml

```diff
@@ -392,9 +392,7 @@ let trigger_slack_webhooks ~stderr ~old_logdir ~new_logdir conf =
     | _ -> Lwt_io.write_line stderr ("Something when wrong with slack webhook "^Uri.to_string webhook)
   end
 
-let get_cap ~stderr =
-  let home = Sys.getenv "HOME" in
-  let cap_file = home^"/ocluster.cap" in (* TODO: fix that *)
+let get_cap ~stderr cap_file =
```

Comment:
```suggestion
let get_cap ~stderr ~cap_file =
```
I prefer using labled parameters when dealing with non-abstract types (like string)

Reply:
Ok, I've done that everywhere else and I forgot this one!